src/components/volume/volume.js

```diff
@@ -183,11 +183,11 @@ class Volume extends Component {
   /**
    * on volume control button click, toggle mute in player and store state
    *
-   * @method onVolumeControlButtonClick
+   * @method VolumeButtonClicked
    * @returns {void}
    * @memberof Volume
    */
-  onVolumeControlButtonClick(): void {
+  VolumeButtonClicked(): void {
```

Comment:
```suggestion
  onVolumeButtonClicked(): void {
```

As it is a handler keep the onX convention and make sure class methods start with lower case